themes/bootstrap3/templates/browse/home.phtml

```diff
@@ -15,7 +15,7 @@
     <?php foreach ($this->browseOptions as $item => $currentOption): ?>
       <a href="<?=$this->url('browse-' . strtolower($currentOption['action'])); ?>" class="browse-item<?php if($currentOption['action'] == $this->currentAction): ?> active<?php endif; ?>">
         <?=$this->transEsc($currentOption['description']) ?>
-        <span class="pull-right flip"><i class="fa fa-angle-right" title="<?=$this->transEscAttr('more') ?>"></i></span>
+        <span class="pull-right flip"><?=$this->icon('more', ['title' => $this->transEscAttr('more')]) ?></span>
```

Comment:
I think we need to change all the transEscAttr() calls to simply translate() when passing them to the icon helper, because the icon helper is doing the escaping, and we don't want to double-escape anything.